### PR TITLE
Add IFNULL function for postgres

### PR DESCRIPTION
**Pull request by aleksbrgt**

### CODE REVIEW

File: config/postgres.yml [hunk 1]
```diff
@@ -5,9 +5,13 @@ doctrine:
...
+
             string_functions:
                 str_to_date: DoctrineExtensions\Query\Postgresql\StrToDate
                 count_filter: DoctrineExtensions\Query\Postgresql\CountFilterFunction
                 string_agg: DoctrineExtensions\Query\Postgresql\StringAgg
                 greatest: DoctrineExtensions\Query\Postgresql\Greatest
                 least: DoctrineExtensions\Query\Postgresql\Least
+
+            numeric_functions:
+                ifnull: DoctrineExtensions\Query\Postgresql\IfNull
```
Comment (IonBazan) [inline config/postgres.yml]: Shouldn't it be in `string_functions`? It is there for MySQL and SQLite.

File: src/Query/Postgresql/IfNull.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    /**
+     * @param SqlWalker $sqlWalker
+     *
+     * @return string
+     */
+    public function getSql(SqlWalker $sqlWalker): string
+    {
+        return sprintf(
+            'IFNULL(%s, %s)',
```
Comment (IonBazan) [inline src/Query/Postgresql/IfNull.php]: I think PostgreSQL does not have `IFNULL` function. Did you mean [`COALESCE`](https://www.postgresql.org/docs/9.3/functions-conditional.html#FUNCTIONS-COALESCE-NVL-IFNULL)?

**Comment by stevelacey** [comment]
@aleksbrgt as @IonBazan said, I think you're looking for COALESCE, and it looks like that should work already in Doctrine without the need for DoctrineExtensions

File: src/Query/Postgresql/IfNull.php (hunk 2 above)
Comment (aleksbrgt) [reply src/Query/Postgresql/IfNull.php]: Postgres has the function https://www.postgresql.org/docs/9.2/functions-conditional.html

**Pull request closed without merging**